remotewin.sh

```diff
@@ -1,5 +1,24 @@
 #!/bin/bash
 
+
+# List of required commands
+required_commands=("xfreerdp" "awk" "grep" "sed")
+
+# Function to check if a command is installed
+check_command() {
```

Comment:
A check for the package 'xfreerdp' is already performed on line 77 (now line 96). Please replace the old check with this one.